src/praisonai/tests/unit/bots/test_outbox_reconcile.py

```
@@ -0,0 +1,198 @@
+        q1 = OutboundQueue(path=str(path))
+        await _seed_in_flight(q1)
+
+        q2 = OutboundQueue(path=str(path))
+
+        sent = []
+
+        async def sender(target, payload):
+            sent.append((target, payload))
+            return True
+
+        succeeded, failed = await q2.drain(sender)
```

Ruff (0.15.18)

[warning] Unpacked variable `failed` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)